app/policies/submission_policy.rb

```diff
@@ -8,7 +8,7 @@ def create?
     everyone
   end
 
-  [:download?, :download_file?, :render_file?, :run?, :score?, :show?, :statistics?, :stop?, :test?].each do |action|
+  [:download?, :download_file?, :render_file?, :run?, :score?, :show?, :statistics?, :stop?, :test?, :insights?].each do |action|
```

Comment:
Which method are you referring here? I can't find `submission_controller#insights`.

Reply:
Alright, I added a comment about that. Thanks!